demo-drone-app/src/components/Drones.vue

```diff
@@ -122,7 +122,37 @@
             </template>
             <template slot="expand" slot-scope="props">
               <v-card flat>
-                <v-card-text>{{props.item.description}}</v-card-text>
+                <v-card-text v-if="!editDescription">
+                  <v-layout row justify-space-between>
+                    <v-flex xs8>
```

Comment:
any reason for xs8? the rest of the structure does not work for that small of a window so it does not really make sense to have that flag there. 

sorry to be nitpicky! my team at work rn is super, super nitpicky but it's made me a ton better so I want to translate that over to this project :D